index.js

```diff
@@ -1,18 +1,18 @@
 'use strict';
 
 module.exports = {
-  extends: [
-    'stylelint-config-standard',
-    // Needed to be able to parse vue files, see https://github.com/ota-meshi/stylelint-config-html#book-usage
-    'stylelint-config-html/vue'
-  ],
   overrides: [
     {
       // Needed to be able to parse scss files, as we don't want to use the stylelint-config-standard-scss
       // see https://stylelint.io/migration-guide/to-14/
       files: '**/*.scss',
       customSyntax: "postcss-scss",
     },
+    {
+      "files": ["*.css"],
+      // 'stylelint-config-html/vue: Needed to be able to parse vue files, see https://github.com/ota-meshi/stylelint-config-html#book-usage
+      "extends": [ 'stylelint-config-standard', 'stylelint-config-html/vue']
+    }
```

Comment:
question: did you test this against a local project (using `npm link`)? Since we use SCSS in Vue, I'm not sure if `stylelint-config-html/vue` is in the right place here... 🤔

In this section it says, that moving the `extends` is only required, if you previously did overwrite them (which we did not, because we only had 1 `extends`): https://github.com/stylelint/stylelint/blob/main/docs/migration-guide/to-15.md#changed-overridesextends-behavior

Reply:
I did use it in a local project and it seems to work, but maybe I missed something. What do you mean with we only had 1 extends?